apps/retrospective/services/sprint_sync.go

```diff
@@ -380,6 +380,48 @@ func (service SprintService) GetTimeTrackerData(sprint retroModels.Sprint, taskP
 	return timeTrackerTaskKeys, sprintMemberTimeLogs, nil
 }
 
+// GetSprintMemberSanitizedTimeTrackerData ...
+func (service SprintService) GetSprintMemberSanitizedTimeTrackerData(
+	taskProviderConfig []byte,
+	timeTrackerConfig []byte,
+	sprint retroModels.Sprint) ([]string, []timeTrackerSerializers.TimeLog, error) {
+
+	_, timeLogs, err := service.GetSprintMemberTimeTrackerData(timeTrackerConfig, sprint)
+	if err != nil {
+		utils.LogToSentry(err)
+		service.SetSyncFailed(sprint.ID)
+		return nil, nil, err
+	}
+	timeLogs, err = tasktracker.SanitizeTimeLogs(taskProviderConfig, timeLogs)
+	if err != nil {
+		utils.LogToSentry(err)
+		service.SetSyncFailed(sprint.ID)
+		return nil, nil, err
+	}
+
+	timeLogsKeymap := map[string]timeTrackerSerializers.TimeLog{}
```

Comment:
This can be optimized further and please add more comments!!

Reply:
done